app/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@auth0/auth0-react": "^2.2.4",
+    "@types/lodash": "^4.17.13",
```

Comment:
the only lodash function we're using at the moment is `isEqual`, and as far as i can tell the bundler is doing a great job tree-shaking it.

i don't know what state the `lodash` project is in. i think v5 has been in the works for a very long time...